errors.js

```diff
@@ -0,0 +1,13 @@
+
+// Handle Custom Errors.
+exports.handleCustomErrors = (err, req, res, next) => {
```

Comment:
We don't have any custom errors in the code yet that would need to be handled - please remove this custom error handler until it is needed.